hardware_accelerators/gaudi-ai-accelerator.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="gaudi-ai-accelerator"]
+= Intel Gaudi AI accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gaudi-ai-accelerator
+
+toc::[]
+
+:FeatureName: Intel Gaudi AI accelerators
+
+You can use Intel Gaudi AI accelerators for your {product-title} generative AI and machine learning (AI/ML) applications. Intel Gaudi AI accelerators offer a cost-efficient, flexible, and scalable solution for optimized deep learning workloads.
+
+Red{nbsp}Hat supports Intel Gaudi 2 and Intel Gaudi 3 devices. Intel Gaudi 3 devices provide significant improvements in training speed and energy efficiency.
+
+include::modules/gaudi-ai-accelerator-prerequisites.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For information on installing Intel Gaudi Base Operator on OpenShift, see link:https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/OpenShift_Installation/index.html[OpenShift Installation].
+
+* For information on integrating Intel Gaudi AI accelerators into OpenShift AI, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.20/html/working_with_accelerators/intel-gaudi-ai-accelerator-integration_accelerators#intel-gaudi-ai-accelerator-integration_accelerators[Intel Gaudi AI Accelerator integration].
+
+
+
+
```

Comment:
Not necessary but these extra lines can be trimmed off

Reply:
deleted lines.